doc/source/conf.py

```diff
@@ -86,6 +86,20 @@
     # type, unless multiple values are being returned"
 }
 
+# Ignoring numpydoc validation on built-in methods from Python
+# or other libraries due to inheritance
+numpydoc_validation_exclude = {
+    "denominator",
+    "imag",
+    "numerator",
+    "real",
+    "__cause__",
+    "__context__",
+    "__abs__",
+    "__float__",
+    "__int__",
+}
+
```

Comment:
When switching the main Python version from 3.12 to 3.13, many numpydoc validation warnings started to appear. Since, by default, we fail on warnings with our doc-build action, I took the opportunity to fix it. Also happened in other libraries. Here we are ignoring the validation on some properties and dunder methods belonging to parent classes (outside of the scope of this library).

Reply:
Is it because numpydoc validation is more sensitive/accurate when tracking down issues?